packages/xo-server-test/src/_xoConnection.js

```diff
@@ -110,8 +110,32 @@ class XoConnection extends Xo {
     return id
   }
 
-  async createTempBackupNgJob(params) {
-    const job = await this.call('backupNg.createJob', params)
+  async createTempBackupNgJob({
+    settings: { '': globalSetting, ...settings } = {},
+    ...params
+  } = {}) {
+    const job = await this.call('backupNg.createJob', {
+      mode: 'full',
+      name: 'default-backupNg',
```

Comment:
I think you could find better default names, e.g. `xo-server-test 2019-04-12T07:04:42`.

Reply:
The job name should be static because we use snapshots.